libraries/redcore/layouts/searchtools/default.php

```diff
@@ -23,7 +23,7 @@
 	'orderFieldSelector'  => '#list_fullordering'
 );
 
-$data['options'] = array_unique(array_merge($customOptions, $data['options']));
+$data['options'] = array_replace($customOptions, $data['options']);
```

Comment:
array_merge() should work. See:
https://github.com/joomla/joomla-cms/blob/staging/layouts/joomla/searchtools/default.php#L25